### ISSUE TITLE
aws-iam-authenticator v0.5.2 release not at expected download URL

### ISSUE THREAD

**Issue by javelinco** [CircleCI-Public/aws-eks-orb#32]
### Orb version
1.0.0

### What happened
- [CircleCI Link (not sure if accessible)](https://app.circleci.com/pipelines/github/OpenSesame/course-authoring-api-course-manager/630/workflows/b3b0cc4f-98ba-42ae-be77-eaeed18daa8f/jobs/1480)

- When running step "Install the AWS IAM Authenticator for Kubernetes" from the "update-kubeconfig-with-authenticator" command, I got the following error:

`curl: (22) The requested URL returned error: 404 Not Found`

After further investigation, I found that the latest v0.5.2 release, which came out 2 hours previously, doesn't have any content at the download url that would be constructed from this script. If I pin the version to v0.5.1, things work properly.

### Expected behavior
- I shouldn't need to pin the version to have this code work - it should have installed the AWS IAM Authenticator and finished the job.

**Comment by dsayling** [comment]
@javelinco It looks like the latest GitHub release did not contain the executable assets as previous GitHub releases did. I opened an issue with the upstream source.

https://github.com/kubernetes-sigs/aws-iam-authenticator/issues/344

If for some reason they choose to move the assets, we can update the orb as necessary.

**Comment by javelinco** [comment]
Thank you! I appreciate you jumping on this so quickly.

**Comment by DFilipeS** [comment]
How can I fix the version for `aws-iam-authenticator`?

**Comment by lokst** [comment]
@DFilipeS You can use the `authenticator-release-tag` parameter and set it to v0.5.1

**Comment by crak-mmarcoux** [comment]
@dsayling Hey Drew,

Can you, while the issue is happening,put a default in the orb to use v0.5.1 for everyone instead of a blank default?
That would unlock everyone until fixed.

If not to hard, shabby for you.

We have over 180 repos, it'd be a pain to update all of them for this :/

**Comment by lokst** [comment]
@crak-mmarcoux We're working on making a patch release of the orb to use v0.5.1 when no version is specified

**Comment by a7i** [comment]
> How can I pin the version for `aws-iam-authenticator`?

This is how I have pinned my build:

```
- aws-eks/update-kubeconfig-with-authenticator:
          authenticator-release-tag: v0.5.1
          cluster-name: << parameters.eks-cluster-name >>
          aws-region: << parameters.region >>
          install-kubectl: true
```

**Comment by JeroenRijks** [comment]
Thanks for the quick fix, @lokst !

**Comment by lcfumes** [comment]
> > How can I pin the version for `aws-iam-authenticator`?
> 
> This is how I have pinned my build:
> 
> ```
> - aws-eks/update-kubeconfig-with-authenticator:
>      authenticator-release-tag: v0.5.1
>      cluster-name: << parameters.eks-cluster-name >>
>      aws-region: << parameters.region >>
>      install-kubectl: true
> ```

It worked perfect for me. Thank you!

**Comment by lcfumes** [comment]
> > How can I pin the version for `aws-iam-authenticator`?
> 
> This is how I have pinned my build:
> 
> ```
> - aws-eks/update-kubeconfig-with-authenticator:
>      authenticator-release-tag: v0.5.1
>      cluster-name: << parameters.eks-cluster-name >>
>      aws-region: << parameters.region >>
>      install-kubectl: true
> ```

It worked perfect to me. Thank you!

**Comment by JeroenRijks** [comment]
The error message in the pipeline says 
```[✖]  creating CloudFormation stack "eksctl-cci-orb-test-defaults-cluster": AlreadyExistsException: Stack [eksctl-cci-orb-test-defaults-cluster] already exists
```
Can someone who has access to this AWS account investigate whether that stack should exist, or whether it should have been deleted by a previous workflow or something?

**Comment by lokst** [comment]
@JeroenRijks We investigated the failing tests, it is related to a vpc limit error from a previous workflow (https://app.circleci.com/pipelines/github/CircleCI-Public/aws-eks-orb/157/workflows/a50da9e4-8e28-41cc-a23c-605c835e6419/jobs/1690/parallel-runs/0/steps/0-106) which unfortunately will take us some time to resolve with AWS. We have done some manual testing to supplement the rest of the integration tests, so the release should be fine.

**Comment by lokst** [comment]
The workaround to use aws-iam-authenticator version v0.5.1 when the version is not specified is available in `circleci/aws-eks@1.0.1`

**Comment by lokst** [comment]
Closing out the issue as https://github.com/kubernetes-sigs/aws-iam-authenticator/issues/344 has been resolved
